index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="style.css" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio setup and mobile</title>
+</head>
+<body>
+       
+      
+       <section class="headline">
+       
+        <header class="toolbar">
+            <a href="#" ><p>My logo </p></a>
+            <div class ="vector">
+                <a href="#" >   <img src = "/Icons/Union.png"> 
+                </a>
+            </div>
```

Comment:
- [ ] Kindly wrap your hamburger button inside the nav tag. This is for semantic reasons.